src/main/java/shop/gitit/member/domain/goods/Goods.java

```diff
@@ -0,0 +1,30 @@
+package shop.gitit.member.domain.goods;
+
+import lombok.Builder;
+import lombok.Getter;
+import org.springframework.data.mongodb.core.mapping.Field;
+import shop.gitit.member.exception.PointViolationException;
+
+@Getter
+public class Goods {
+    private static final int ZERO = 0;
+
+    @Field(name = "member_commit_count")
+    private int point;
+
+    @Builder
+    public Goods(int point) {
+        this.point = point;
+    }
```

Comment:
```suggestion
@Getter
// 1 @NoArgsConstructor(access~~) 추가해줘야 함
public class Goods {
    private static final int ZERO = 0;

    @Field(name = "member_commit_count")  // 2
    private int point;

    @Builder
    public Goods(int point) {
        this.point = ZERO;  // 4
    }
 ```
1. 기본 생성자 protect로 막아줘야 될거 같아
2. 포인트 필드의 칼럼명이 잘못된거 같은데?
3. 도메인 명이 Goods인데 처음 봤을 때 Goods가 뭔지 용어의 의미가 바로 파악이 안되는거 같아서 다른 이름으로 하는게 어떤가
4. 최초 생성시 포인트는 0포인트로 시작해야 하니 ZERO로 초기화를 해줘야 하는거 아닌가?

Reply:
1번, 2번은 수정 했어
3번은 고민 했었는데 Goods 대신 할 만한 용어가 보유 자산이나 재화의 의미를 담는 명확하게 들어맞는 표현이 Goods이더라구 currency, asset, source 정도 고민 했었는데 괜찮은 용어 있으면 추천해줘
4번은 테스트 코드 작성하다가 수정한 내용인데 초기 데이터를 직접 넣고 싶어서 초기화 값을 입력 받는 형태로 했거든 만약에 zero로 초기화 하면 테스트할 때 원하는 값을 넣는 방법을 쓸 때 포인트를 넣어주는 다른 메서드를 호출해야해서 단위테스트의 목적에 어긋나더라구 어떻게 해결하면 좋을까?

Reply:
4번은 테스트 할 때, 기본 객체를 만들고 나서 포인트를 추가시키는 메서드를 호출하는 방식으로 할 수 있게 만드는게 나을 거같은데?
지금 코드는 약간 테스트를 위한 코드같애

Reply:
오케이 그럼 기본 객체 생성하도록 수정할게